packages/manager/modules/hub/src/components/user-panel/template.html

```diff
@@ -56,7 +62,12 @@
             <span data-translate="hub_links_tasks"></span>
         </a>
         <span class="manager-hub-user-panel_divider" aria-hidden="true"></span>
-        <a data-ng-href="{{ $ctrl.RedirectionService.getURL('support') }}">
+        <a
+            data-ng-href="{{ :: $ctrl.RedirectionService.getURL('support') }}"
```

Comment:
This looks like the logic is quite the same for all links (a link, a tracking name, an icon) maybe we could handle it with a `ng-repeat` ? (This can be done in another iteration)

Reply:
i'll leave it like that for now because of the chatbot button in the middle of the links, i'm not sure how to refactor that cleanly